styles.css

```diff
@@ -96,6 +96,22 @@ h3 small {
 }
 
 
+.twitter-timeline-rendered {
+  margin: auto;
+  margin-bottom: 25px !important;
```

Comment:
no worries. Sometimes you have to do this when dealing with third party stuff. That's life :-)